project/DeployPlugin.scala

```diff
@@ -18,8 +18,8 @@ object DeployPlugin extends AutoPlugin {
       println(rmi)
 
       // Delete pods associated with this deployment, so they spin back up with the new version
-      val delete = Process("kubectl delete pods -l service=" + baseDirectory.value.getName().replace("magda-", "")).!!
-      println(delete)
+      // val delete = Process("kubectl delete pods -l service=" + baseDirectory.value.getName().replace("magda-", "")).!!
```

Comment:
Did you mean to leave this commented? Do we not want to do this any more?

Reply:
Yeah, I meant to comment it.  I was making the behavior of the Scala docker build consistent with the TypeScript.  I could have gone the other way (make the TypeScript restart the pods, too), but I haven't found the behavior to be particularly useful.  Have you?

Reply:
Ok cool